src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Altinn.App.Api.csproj

```diff
@@ -1,18 +1,18 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <LangVersion>latest</LangVersion>
-    <Version>1.3.0</Version>
+    <Version>1.3.1</Version>
```

Comment:
this needs to be bumped further

Reply:
I initially bumped the versions just in case, but I don't think there is a need to create new packages based on this PR. These changes will be included once new versions are needed.